docs/7.x/selinux.md

```diff
@@ -0,0 +1,266 @@
+
+Kubernetes supports SELinux with [seLinuxOptions](https://v1-17.docs.kubernetes.io/docs/tasks/configure-pod-container/security-context/#assign-selinux-labels-to-a-container) inside the [securityContext](https://v1-17.docs.kubernetes.io/docs/tasks/configure-pod-container/security-context/). The options can be configured either for the whole Pod or per-container:
+
+```yaml
+...
+securityContext:
+  seLinuxOptions:
+    type: "my_type_t"
+    level: "s0:c123,c456"
+```
+
+Level specifies the SELinux MCS/MLS security level the container (or containers if specified for the Pod) is run with.
+If left unspecified, container runtime (Docker in this case) will generate a unique level.
+
+The security context will also apply to Pod's volumes (where applicable). The volumes that support SELinux labeling,
+are automatically relabeled according to the specified level:
+
+  * If the `level` is left unspecified, the container runtime will use the same level generated for the container to label the volume
+  * Otherwise, the volume is labeled with the specified level
+
+If multiple Pods run with the same `level`, they will be able to share volumes. If you need inter-Pod protection, consider running
+the Pods with unique MCS `level` or leave the `level` unspecified so it gets automatically generated to be unique.
+
+Kubernetes groups volume storage types into groups and differentiates between Unshared and Shared. SELinux labeling will differ depending on the storage type group.
+`Secret` (and consequently all `EmptyDir`-derived volume types) are examples of Unshared storage and support SELinux labeling, while `hostPath` (as a Shared storage) does not support labeling at the time of this writing.
+
+If you don't specify SELinux domain (`seLinuxOptions.type`) or leave `seLinuxOptions` unspecified, containers will run in the default container process domain.
+
+One implication of the SELinux labeling concerns the volumes bind-mounted from host inside the Planet container.
+In order for external volumes to be accessible to Kubernetes workloads they need to be labeled with a label accessible to the process domain the
+containers are run with.
+By default, each volume will automatically get relabeled at install time to the default container file label unless overridden.
+The label can be customized in the application manifest:
+
+```yaml
+nodeProfiles:
+  - name: node
+    ...
+    volumes:
+      - path: /var/data
+        targetPath: /var/data
+        seLinuxLabel: "system_u:object_r:my_file_type_t:s0"
+```
+
+If the directory has been labeled prior to the installation and relabeling is undesired, the labeling can be turned off with:
+
+```yaml
+nodeProfiles:
+  - name: node
+    ...
+    volumes:
+      - path: /var/data
+        targetPath: /var/data
+        seLinuxLabel: none
+```
+
+
+!!! warning "Performance"
+  Relabeling of directories with a large number of files/sub-directories can be time-consuming.
+  This will be improved in future releases.
```

Comment:
I would avoid making promises about future work in the docs.  We can't predict what our priorities will be.